board/jungle/scripts/reset_test.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
```

Comment:
this is a good test to evaluate this change, but I don't think it has value going forward. I'd remove it